openadapt/window/_windows.py

```diff
@@ -87,9 +84,10 @@ def get_active_element_state(x: int, y: int):
     return properties
 
 
-def get_active_window(depth=10, max_width=10, filename=None) -> Desktop:
-    """
-    Get the active window object.
+def get_active_window(
+    depth: int = 10, max_width: int = 10, filename: str = None
+) -> Desktop:
```

Comment:
should be pywinauto.application.WindowSpecification

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/6159a69dd7340dd520bd020a57101dfd9d444c9f